docs/installation.md

```diff
@@ -0,0 +1,55 @@
+# Installation
+
+These instructions are likely in flux. If something looks out of date or is not working, please raise an issue.
+
+## Install dependencies
+
+Until a conda package is live, the core dependencies are listed in `devtools/conda-envs/minimal_env.yaml` and optional dependencies can be found in `devtools/conda-envs/test_env.yaml`.
```

Comment:
For me, atleast with the new changes, environment needed also looking at `.github/workflows/ci.yaml`.